.github/workflows/bdd.yml

```diff
@@ -2,7 +2,8 @@
 name: BDD
 
 on:
-  deployment_status
+  deployment_status:
+  workflow_dispatch:
```

Comment:
Please revert this to the original version that only contains `deployment_status`.